src/components/organisms/UiMultipleChoices/_internal/UiMultipleChoicesItem.vue

```diff
@@ -349,6 +354,14 @@ const optionsToRender = computed(() => props.options.map((option) => ({
     }
   }
 
+  &__label {
+    font: functions.var($element + "-label", font, var(--font-body-1-thick));
+
+    @include mixins.from-tablet {
+      font: functions.var($element + "-tablet-label", font, var(--font-body-1));
+    }
+  }
+
```

Comment:
```suggestion
  &__label {
    font: functions.var($element + "-label", font, var(--font-body-1-thick));

    @include mixins.from-tablet {
      font: functions.var($element + "-tablet-label", font, var(--font-body-1));
    }
  }
```
Please remove this changes from this PR :thank_you: